less/index.less

```diff
@@ -57,3 +58,271 @@ html, body {
     height: 100%;
 }
 
+//Theme Variables
+@secondary-color: @apricot; 
+
+//Color 
+@apricot: #ED8761;
+
+//Mixin
+.center(@direction, @justify, @align) {
+	display: flex;
+	flex-direction: @direction; 
+	justify-content: @justify;
+	align-items: @align; 
+}
+
+.progress(@percent) {
```

Comment:
Awesome job getting the progress bar to work. This is some advanced creative stuff!